### DISCUSSION TITLE
Cannot delete translations

### DISCUSSION THREAD

**Discussion by gkielwasser** [directus/directus#8732] [Archive]
Hi,

I have added translations relation on a collection using the default workflow.

I have configured that on Deselect translations it should trigger a delete of the translation
![Capture](https://user-images.githubusercontent.com/1532438/136989687-d7d27b88-c4cc-4ff6-be93-f17a4d22607e.PNG)
n

What is happening: when I delete the translations using the "clear value" interface, it Patch the translations with undefined
```
PATCH items/administratives_areas/6b2dea83-70fe-49fc-b5a5-9371a67ad8bb
{
    "translations": null
}
```
But it doesn't delete the translations and they are returned back.
Patch response
```
{
  "id": "6b2dea83-70fe-49fc-b5a5-9371a67ad8bb",
  "parent": null,
  "short_code": "fr",
  "translations": [
    28,
    29,
    30,
    31
  ],
  "children": []
}
```

Thanks

**Comment by rijkvanzanten** [comment]
For any (potential) bugs, please use [Issues](https://github.com/directus/directus/issues/new?assignees=&labels=Bug+%28Potential%29&template=bug_report.yml) 👍🏻

**Reply by gkielwasser** [reply]
Here is the [linked issue](https://github.com/directus/directus/issues/8738)
